src/Expander/Monad.hs

```diff
@@ -507,7 +521,7 @@ getDecl ptr =
     Just decl -> flattenDecl decl
   where
     flattenDecl ::
```

Comment:
Arguably, it is `getBody` above which is flattening a declaration tree into a flat list of declarations. Perhaps `completeDecl` would be a better name than `flattenDecl`?

Reply:
My only concern about `complete` is that it can also be an adjective. What about "finalizeDecl" instead?

Reply:
Wait a minute, we already have a specialized verb for what `flattenDecl` is doing! `flattenDecl` is replacing a bunch of `Ptr`s with their target. So `zonkDecl`?

Reply:
renamed to zonkDecl